From 73ea41b3622e2e578d928f7513941aac9d873279 Mon Sep 17 00:00:00 2001
From: Ben Gamari <ben@smart-cactus.org>
Date: Mon, 1 Jul 2019 11:02:45 -0400
Subject: [PATCH] Fix incorrect case fallthrough

The error message lookup logic would fallthrough from the
forkSetuidFailed case into the default case, meaning that the error
message of the former would never be returned.
---
 cbits/runProcess.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cbits/runProcess.c b/cbits/runProcess.c
index c621158..10794bc 100644
--- a/cbits/runProcess.c
+++ b/cbits/runProcess.c
@@ -329,6 +329,7 @@ runInteractiveProcess (char *const args[],
             break;
         case forkSetuidFailed:
             *failed_doing = "runInteractiveProcess: setuid";
+            break;
         default:
             *failed_doing = "runInteractiveProcess: unknown";
             break;
